docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,503 @@
+---
+title: "Plant Watering Robot with a Raspberry Pi"
+linkTitle: "Watering Robot"
+weight: 50
+type: "docs"
+description: "Create a plant watering robot with a Raspberry Pi single-board computer."
+tags: ["raspberry pi", "app", "board", "motor"]
+# SMES: Olivia Miller
+---
+
+![Picture of the plant watering robot on a desk.](../../img/plant-watering-pi/plant-watering-robot.png)
+
+Building a useful robot doesn't have to require complicated code or expensive equipment.
+With a Raspberry Pi and some cheap, basic hardware, you can keep your plants healthy and happy from anywhere in the world!
+
+Follow this tutorial to learn how to set up an automatic plant watering system:
+
+1.  [Complete the physical assembly and wiring](/#set-up-your-plant-watering-robot).
+2. [Create and connect to the robot](#configure-the-components-of-your-robot-in-the-viam-app).
+3. [Configure your robot's components](#configure-the-components-of-your-robot-in-the-viam-app).
+4. [Configure a custom Capacitive Soil Moisture sensor component](#configure-the-capacitive-soil-moisture-sensor-as-a-custom-sensor-component-model).
+5. [Write code utilizing the Viam Python SDK to control the plant watering robot](#add-python-control-code).
+
+The tutorial uses the following hardware, but you can adjust it as needed:
+
+- A Raspberry Pi with SD card
```

Comment:
```suggestion
- A Raspberry Pi 3B or 4B with SD card
```
Since others won't currently work - I think actually a W 2 Zero will - but the instructions here are for the pin layout etc of the 3B/4B